### DISCUSSION TITLE
minio metrics query

### DISCUSSION THREAD

**Discussion by rbs-cloud** [minio/minio#13592] [Q&A] [answered]
HI Team,
Is there any metric that support to STS request(Number of successful and failure STS requests ) metrics like s3(Total number S3 requests) metrics or planning in future if does not support currently.
along with that we need to know below metrics also supported by minio if yes what could be name of the metrics names from below link
https://github.com/minio/minio/blob/master/docs/metrics/prometheus/list.md

1) Time to process each File access request
2)Volume of data transferred via S3 interface

3)Number of concurrent requests processed via S3 interface

**Comment by klauspost** [comment, marked as answer]
@rbs-cloud If you are interested in a roadmap or possible features, write `hello@minio.io`. 

We might pick up community requests, otherwise you are welcome to contribute a pull request.
